nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -231,6 +231,38 @@ define([
          * extra functions that the Narrative can call.
          */
         function specializeCell() {
+
+            // minimizes the cell
+            cell.minimize = function() {
```

Comment:
Genius copy-paste work here!

Reply:
Can we extract this so it isn't copy paste? It can be a shared cell component code in `common/cellComponents`, that way all the cells can share the same code and there isn't duplicate code. 

Once it is a shared component, it would be nice for it to be tested in it's own spec file.

Reply:
Yeah, I put it into the refactor ticket as I am not sure how we stand for Monday's demo, but it should be easy to do now.